qiita_pet/templates/artifact_ajax/artifact_summary.html

```diff
@@ -111,10 +111,16 @@
 <div class='row'>
   <div class='col-md-12'>
     <h4>
-      {% if processing_info and not processing_info['software_active'] %}
-        <div class="alert alert-warning" role="alert">
-          Warning, the software that generated this artifact is deprecated as a new version is available.
-        </div>
+      {% if processing_info %}
+        {% if processing_info['software_deprecated'] %}
+          <div class="alert alert-danger" role="alert">
+            Danger, the software that generated this artifact was produced by a software version with a known bug and the results are wrong, please re-run with the newer version.
```

Comment:
Danger, the software that generated this artifact was produced by a software version with a known bug and the results are wrong, please re-run with the newer version. -> Warning, the software that generated this artifact has a known bug, and these results are incorrect. Please re-run to produce new results with the latest version of the software.